### ISSUE TITLE
'data_files' is not defined

### ISSUE THREAD

**Issue by abruski** [ZoomerAnalytics/xlwings#155]
I get this error during installation (pip & python setup.py install): 
```
Traceback (most recent call last):
  File "setup.py", line 32, in <module>
    data_files=data_files,
NameError: name 'data_files' is not defined
```
. Any suggestions will be useful.

**Comment by fzumstein** [comment]
How are you installing xlwings? `pip install xlwings`?

**Comment by cel4** [comment]
See also this issue on stackoverflow:
http://stackoverflow.com/questions/28689673/installation-of-xlwings-using-pip-and-from-source-gives-an-error-in-ubuntu-vm

**Comment by fzumstein** [comment]
Are you guys trying to install xlwings on Linux? At this time, only Windows and Mac are supported.

**Comment by cel4** [comment]
This is probably the issue. I would suggest adding an `else` block and throw a error message in case of an unsupported OS. The name error is quite obscure.

**Comment by cel4** [comment]
I think this tiny modification already makes the actual error clearer: https://github.com/ZoomerAnalytics/xlwings/pull/157

**Comment by fzumstein** [comment]
Implemented with 0.3.3
